.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: python
 python: 2.7
+dist: trusty
+sudo: required
```

Comment:
This is the new default, so shouldn't be necessary.

Reply:
Ah ok. Did not know. Thanks